src/js/models/messages.js

```diff
@@ -39,11 +42,12 @@ export default class Messages{
 			if (this.unsubscribe) {
 				this.unsubscribe();
 			}
-			this.unsubscribe = db.collection('Messages').doc(chatId).collection('history').orderBy('time').onSnapshot(snapshot => {
+			this.unsubscribe = await db.collection('Messages').doc(chatId).collection('history').orderBy('time').onSnapshot(snapshot => {
```

Comment:
This returns a function, nothing to await
```suggestion
			this.unsubscribe = db.collection('Messages').doc(chatId).collection('history').orderBy('time').onSnapshot(snapshot => {
```